requirements.txt

```diff
@@ -4,5 +4,8 @@ Sphinx
 sphinx-rtd-theme
 sphinxcontrib-spelling
 sphinxcontrib-websupport
+sphinx-copybutton
+sphinx-togglebutton
+sphinx-panels
```

Comment:
I found that `sphinx-panels` is not compatible with this theme.

https://github.com/executablebooks/sphinx-panels/issues/57

We can upvote it to get more attention to the issue.

`sphinx-togglebutton` is fine for what we need.